### PR TITLE
Add new cryptocurrencies

### PR DESCRIPTION
**Pull request by IgnacioPrez**

### CODE REVIEW

**Comment by IgnacioPrez** [comment]
feature3

feature1

File: src/components/card/Card.tsx [hunk 1]
```diff
@@ -16,16 +16,14 @@ import {
...
   const formatter = new Intl.NumberFormat('en-US', {
     style: 'currency',
     currency: 'USD',
     minimumFractionDigits: 0,
   });
 
-  const roundNumber =
-    metrics.market_data.percent_change_btc_last_24_hours.toFixed(2);
-
+  const roundNumber = market_data.percent_change_btc_last_24_hours.toFixed(2);
```
Comment (tomasdematos) [inline src/components/card/Card.tsx]: use useMemo, to prevent redeclarations

File: src/components/header-back/HeaderBack.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Pressable} from 'react-native';
+import {ContainerHeaderScreen, TextLinkAdd} from './styles';
+interface PropsNavigation {
+  navigation: any;
+}
+const HeaderBack = ({navigation}: PropsNavigation) => {
+  return (
```
Comment (tomasdematos) [inline src/components/header-back/HeaderBack.tsx]: use implicit return

File: src/screen/AddScreen.tsx [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
-import {View, Text} from 'react-native';
-const AddScreen = () => {
+import {HeaderBack, Search} from '../components';
+import {ContainerAddScreen} from '../components/screen-style-component/style';
+const AddScreen = ({navigation}: any) => {
```
Comment (tomasdematos) [inline src/screen/AddScreen.tsx]: implicit return

File: src/services/urls.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 export const ImageUrl = 'https://asset-images.messari.io/images/';
 
 export const baseUrl = 'https://data.messari.io/api/v2';
-/*
-https://data.messari.io/api/v2/assets?fields=id,slug,symbol,metrics/market_data&&limit=2 */
+
+export const searchUrl = 'https://data.messari.io/api/v1/assets';
```
Comment (tomasdematos) [inline src/services/urls.ts]: this should be stored in a .env file

https://yarnpkg.com/package/react-native-dotenv

File: src/slices/criptoSlice.ts [hunk 5]
```diff
@@ -34,17 +33,24 @@ export const cryptoSlice = createSlice({
...
+      .addCase(fetchCrypto.pending, state => {
         state.loading = true;
-        state.error = null;
       })
-      .addCase(fetchCryptos.fulfilled, (state, action) => {
+      .addCase(fetchCrypto.fulfilled, (state, action) => {
         state.loading = false;
-        state.crypto = action.payload;
+        const cryptoFound = state.crypto.find(
+          el => el.id === action.payload.id,
```
Comment (tomasdematos) [inline src/slices/criptoSlice.ts]: use more descriptive name, instead of el

**Pull request merged**